src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ViewCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ViewCommand object.
+ */
+public class ViewCommandParser implements Parser<ViewCommand> {
+    public static final String MESSAGE_ONE_BASED_INDEXING =
+            "Please input a 1-based integer.";
```

Comment:
Do you mean index?

Reply:
Yes, thanks for spotting it, Have amended it